cockpit/src/subscriptions-view.jsx

```diff
@@ -20,17 +20,20 @@
 import cockpit from 'cockpit';
 import React from 'react';
 import subscriptionsClient from './subscriptions-client';
-import { ListView, ListViewItem, ListViewIcon } from 'patternfly-react';
-import { Row, Col } from 'react-bootstrap';
 import { InsightsStatus } from './insights.jsx';
 import { EmptyStatePanel } from "../lib/cockpit-components-empty-state.jsx";
-import { ExclamationCircleIcon } from '@patternfly/react-icons';
-
-import './subscriptions-view.scss';
+import { ListingTable } from "../lib/cockpit-components-table.jsx";
+import { ExclamationCircleIcon, OutlinedCheckCircleIcon, ErrorCircleOIcon } from '@patternfly/react-icons';
```

Comment:
`OutlinedCheckCircleIcon`, `ErrorCircleOIcon` are not used in the code.

Reply:
Done